From 093fd99a4c12d902110ac15cd583c37ee6f2ea12 Mon Sep 17 00:00:00 2001
From: David Marchand <david.marchand@redhat.com>
Date: Thu, 3 Oct 2019 20:45:20 +0200
Subject: doc: Fix incorrect reference for dpdk-testpmd.

Move back the dpdk-testpmd reference to the right section of this
document so that the link in howto/dpdk does not point to
"vhost-user-client tx retries config".

Fixes: 080f080c3bc1 ("netdev-dpdk: Enable tx-retries-max config.")

Signed-off-by: David Marchand <david.marchand@redhat.com>
Acked-by: Kevin Traynor <ktraynor@redhat.com>
Signed-off-by: Ilya Maximets <i.maximets@ovn.org>
---
 Documentation/topics/dpdk/vhost-user.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/topics/dpdk/vhost-user.rst b/Documentation/topics/dpdk/vhost-user.rst
index 724aa62f6..fab87bd3d 100644
--- a/Documentation/topics/dpdk/vhost-user.rst
+++ b/Documentation/topics/dpdk/vhost-user.rst
@@ -348,8 +348,6 @@ The default value is ``false``.
     across a 10Gbps link possibly unacceptably slow. So recommended hugepage
     size is 2MB.
 
-.. _dpdk-testpmd:
-
 vhost-user-client tx retries config
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
@@ -375,6 +373,8 @@ Tx retries max can be set for vhost-user-client ports::
 
   Configurable vhost tx retries are not supported with vhost-user ports.
 
+.. _dpdk-testpmd:
+
 DPDK in the Guest
 -----------------
 
-- 
2.14.1


